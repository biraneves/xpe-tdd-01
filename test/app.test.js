const request = require('supertest');
const app = require('../src/app');

describe('Testes de integração', () => {
    test('Responder HTTP 200 na raiz.', () => {
        return request(app)
            .get('/')
            .then((res) => expect(res.status).toBe(200));
    });

    test('Responder HTTP 200 na raiz', async () => {
        const res = await request(app).get('/');
        expect(res.status).toBe(200);
    });
});
