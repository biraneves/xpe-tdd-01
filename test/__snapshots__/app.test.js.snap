// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de integração Cenário 1 1`] = `
{
  "juro": 0.025,
  "montante": 106.9,
  "parcelas": 3,
  "prestacoes": [
    35.64,
    35.63,
    35.63,
  ],
  "primeiraPrestacao": 35.64,
}
`;

exports[`Testes de integração Cenário 2 1`] = `
{
  "juro": 0.025,
  "montante": 106.9,
  "parcelas": 3,
  "prestacoes": [
    35.64,
    35.63,
    35.63,
  ],
  "primeiraPrestacao": 35.64,
}
`;

exports[`Testes de integração Cenário 3 1`] = `
{
  "juro": 0.025,
  "montante": 106.9,
  "parcelas": 3,
  "prestacoes": [
    35.64,
    35.63,
    35.63,
  ],
  "primeiraPrestacao": 35.64,
}
`;
